src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -556,4 +556,9 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
         severity: ChangeSeverity::Info,
         summary: "New option `build.windows-rc` that will override which resource compiler on Windows will be used to compile Rust.",
     },
+    ChangeInfo {
+        change_id: 99999,
```

Comment:
Looks like you accidentally forgot to update this. 😅

Reply:
:facepalm: I'll update it the next time we modify the change tracker, to avoid churn. Thanks!

Reply:
In that case I might as well fix it in https://github.com/rust-lang/rust/pull/146929, assuming another opportunity doesn’t come up first.

Reply:
Ok, let's do that please :innocent: Thanks!